deltachat-contact-tools/src/lib.rs

```diff
@@ -774,6 +804,33 @@ END:VCARD",
         assert_eq!(contacts[0].profile_image, None);
     }
 
+    /// Proton at some point slightly changed the format of their vcards
+    #[test]
+    fn test_protonmail_vcard2() {
+        let contacts = parse_vcard(
+            r"BEGIN:VCARD
+VERSION:4.0
+FN;PREF=1:Alice
+PHOTO;PREF=1:data:image/jpeg;base64\,/9aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+ aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+ aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/Z
+REV:Invalid Date
+ITEM1.EMAIL;PREF=1:alice@example.org
+KEY;PREF=1:data:application/pgp-keys;base64\,xsaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
```suggestion
KEY;PREF=1:data:application/pgp-keys;base64,xsaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

usually Proton exports the vCard without the backslash though, so I guess it shouldn't be there in the test. Or maybe add a second test or second key item to the test?